apps/web/src/common/modules/modals/NotificationEmailModal.vue

```diff
@@ -0,0 +1,276 @@
+import { postValidationCode, postValidationEmail } from '@/lib/helper/verify-email-helper';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+import { useProxyValue } from '@/common/composables/proxy-state';
+
+interface Props {
+    domainId: string
+    userId: string
+    verified: boolean
+    email: string
+    isAdministration: boolean
+    visible: boolean
+}
+
+const props = withDefaults(defineProps<Props>(), {
+    domainId: '',
+    userId: '',
+    verified: false,
+    email: '',
+    isAdministration: false,
+    visible: false,
+});
+
+const vm = getCurrentInstance()?.proxy as Vue;
+
+const emit = defineEmits(['visible', 'handle-user-detail']);
+
+const state = reactive({
+    loading: false,
+    isCollapsed: true,
+    isEditMode: props.verified,
+    myId: computed(() => store.state.user.userId),
+    email: computed(() => (props.isAdministration ? props.email : store.state.user.email)),
+    proxyVisible: useProxyValue('visible', props, emit),
+});
+const formState = reactive({
+    newNotificationEmail: '',
+    verificationCode: '',
+});
+const validationState = reactive({
+    isValidationCodeValid: undefined as undefined | boolean,
+    validationCodeInvalidText: '' as TranslateResult | string,
+});
+
+/* Components */
+const handleEditButton = () => {
+    state.isEditMode = true;
+};
+const handleClickCancel = () => {
+    state.proxyVisible = false;
+    handleReset();
+};
+const handleReset = () => {
+    formState.newNotificationEmail = '';
+    formState.verificationCode = '';
+    validationState.isValidationCodeValid = false;
+    validationState.validationCodeInvalidText = '';
+};
+
+/* API */
+const handleClickSendEmailButton = async (resend?: boolean) => {
+    state.loading = true;
+    try {
+        await postValidationEmail({
+            user_id: props.userId,
+            domain_id: props.domainId,
+            email: resend === true ? state.email : formState.newNotificationEmail,
+        });
+        if (state.myId === props.userId) {
+            if (resend !== true) {
+                await store.dispatch('user/setUser', { emailVerified: false, email: formState.newNotificationEmail });
+            }
+        }
+        emit('handle-user-detail');
+        state.isEditMode = false;
+    } catch (e: any) {
+        ErrorHandler.handleError(e);
+        throw e;
+    } finally {
+        state.loading = false;
+    }
+};
+const handleClickConfirmButton = async () => {
+    state.loading = true;
+    try {
+        await postValidationCode({
+            user_id: props.userId,
+            domain_id: props.domainId,
+            code: formState.verificationCode,
+        }, state.myId === props.userId);
+        emit('handle-user-detail');
+        state.proxyVisible = false;
+        handleReset();
+    } catch (e) {
+        validationState.isValidationCodeValid = true;
+        validationState.validationCodeInvalidText = vm.$t('COMMON.NOTIFICATION_MODAL.INVALID_CODE');
```

Comment:
```suggestion
        validationState.validationCodeInvalidText = i18n.t('COMMON.NOTIFICATION_MODAL.INVALID_CODE');
```
todo: Please use i18n.t() :) Not just here, but everywhere.